### PR TITLE
Add ReactPHP core team as authors to composer.json

### PR DESCRIPTION
**Pull request by WyriHaximus**
This will show the core team as authors under the dependencies on Packagist.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,6 +3,26 @@
...
+            "homepage": "https://clue.engineering/",
+            "email": "christian@clue.engineering"
+        },
+        {
+            "name": "Cees-Jan Kiewiet",
+            "homepage": "https://wyrihaximus.net/",
+            "email": "reactphp@ceesjankiewiet.nl"
+        },
+        {
+            "name": "Jan Sorgalla",
```
Comment (jsor) [inline composer.json:18]:
```suggestion
            "name": "Jan Sorgalla",
            "homepage": "https://sorgalla.com",
```

Comment (WyriHaximus) [reply composer.json:18]: Added it

File: composer.json [hunk 2]
```diff
@@ -3,6 +3,27 @@
...
+            "homepage": "https://sorgalla.com/",
+            "email": "jsorgalla@gmail.com"
+        },
+        {
```
Comment (cboden) [inline composer.json:22]: Looking at my commit history for this repo, I've primary only done peer merges and the odd tooling cleaning. I would not consider myself an author of this repo.

Comment (WyriHaximus) [reply composer.json:22]: @cboden Well the way I look at it, if you are on the team, reviewing code puts you at the same level of responsibility as writing code as by approving the code you say "I trust this code as if I wrote it myself and I take the responsibility for it as if I wrote it myself".

Comment (cboden) [reply composer.json:22]: 🤷‍♂️ ok 😄

**Pull request merged**